aiida/backends/manager.py

```diff
@@ -121,6 +121,11 @@ def load_backend_environment(self, profile, validate_schema=True, **kwargs):
         :param validate_schema: boolean, if True, validate the schema first before loading the environment.
         :param kwargs: keyword arguments that will be passed on to the backend specific scoped session getter function.
         """
+        from aiida.backends.managers.profile_access import ProfileAccessManager
+
+        access_manager = ProfileAccessManager(profile)
+        access_manager.request_access()
+
```

Comment:
As mentioned before, I would rather this did not go here: since I will be removing the whole concept of `BackendManager` in #5172 (instead, backends will just be loaded in the standard `__init__` manner).
This code would then better go in `Manager._load_backend`, where this method is actually called

Reply:
Done